### PR TITLE
Fix top consumer exception

### PR DESCRIPTION
**Pull request by upalatucci**
## 📝 Description

Exception raised one time for `undefined has no method getDropdownLabel`

### CODE REVIEW

File: src/views/clusteroverview/overview/components/top-consumers-card/utils/TopConsumerCard.tsx [hunk 1]
```diff
@@ -52,20 +52,20 @@ const TopConsumerCard: React.FC<TopConsumersMetricCard> = ({ cardID }) => {
             isCheckboxSelectionBadgeHidden
           >
             {TopConsumerMetric.getAll().map((metric) => (
-              <SelectOption key={metric.getValue()} value={t(metric.getDropdownLabel())} />
+              <SelectOption key={metric.getValue()} value={t(metric?.getDropdownLabel())} />
```
Comment (avivtur) [inline src/views/clusteroverview/overview/components/top-consumers-card/utils/TopConsumerCard.tsx:55]: please change metric?.getValue()

File: src/views/clusteroverview/overview/components/top-consumers-card/utils/TopConsumerCard.tsx [hunk 2]
```diff
@@ -52,20 +52,20 @@ const TopConsumerCard: React.FC<TopConsumersMetricCard> = ({ cardID }) => {
...
             ))}
           </FormPFSelect>
         </div>
         <div className="kv-top-consumer-card__scope-select">
           <FormPFSelect
             toggleId="kv-top-consumers-card-scope-select"
             variant={SelectVariant.single}
-            selections={t(TopConsumerScope.fromString(scopeKey).getDropdownLabel())}
+            selections={t(TopConsumerScope.fromString(scopeKey)?.getDropdownLabel())}
```
Comment (avivtur) [inline src/views/clusteroverview/overview/components/top-consumers-card/utils/TopConsumerCard.tsx:63]: is TopConsumerScope guarenteed to be defined? if not add here ? as well

File: src/views/clusteroverview/overview/components/top-consumers-card/utils/TopConsumerCard.tsx [hunk 3]
```diff
@@ -52,20 +52,20 @@ const TopConsumerCard: React.FC<TopConsumersMetricCard> = ({ cardID }) => {
...
             onSelect={(e, value) => onScopeSelect(value)}
             isCheckboxSelectionBadgeHidden
           >
             {TopConsumerScope.getAll().map((scope) => (
-              <SelectOption key={scope.getValue()} value={t(scope.getDropdownLabel())} />
+              <SelectOption key={scope.getValue()} value={t(scope?.getDropdownLabel())} />
```
Comment (avivtur) [inline src/views/clusteroverview/overview/components/top-consumers-card/utils/TopConsumerCard.tsx:68]: here too, change to scope?.getValue()

File: src/views/clusteroverview/overview/components/top-consumers-card/utils/TopConsumerCard.tsx (hunk 2 above)
Comment (upalatucci) [reply src/views/clusteroverview/overview/components/top-consumers-card/utils/TopConsumerCard.tsx:63]: yes. It's a class and fromString is a static method of this class

**Pull request merged**